### ISSUE TITLE
Web Accessibility

### ISSUE THREAD

**Issue by dylakim** [dylakim/portfolio#6]
As a developer, I want to build an accessible website to comply with ADA standards since it's important to make the web accessible to everyone.

Acceptance criteria:
- [x] Passes Axe testing

Developer notes:
- [x] The site must have a page title
- [x] All images should have appropriate alt text
- [x] All links should have appropriate titles
- [x] The site should have an appropriate heading hierarchy
- [x] Text should be resizable
- [x] User should be able to navigate with only the keyboard
- [ ] Forms should have proper labels and error messaging

Creative notes:
- [x] Ensure contrast ratio complies with WCAG standards
